kolibri/core/content/utils/paths.py

```diff
@@ -49,7 +46,15 @@ def get_local_content_storage_file_url(obj):
     The same url will also be exposed by the file serializer.
     """
     if get_attribute(obj, "available"):
-        return get_content_storage_file_url(filename=get_content_file_name(obj))
+        filename = get_content_file_name(obj)
+        return "/{}{}/{}/{}".format(
+            get_content_storage_url(
+                conf.OPTIONS["Deployment"]["URL_PATH_PREFIX"]
+            ).lstrip("/"),
+            filename[0],
+            filename[1],
+            filename,
+        )
```

Comment:
Elsewhere in this file `urljoin` is used, which I think would make this clearer.

At the moment it's confusing what the expected output path structure is, without more context about what the possible values of `URL_PATH_PREFIX` and `get_content_storage_url` are.

(I do see the code was copied to here and predated this refactor)

Reply:
I dimly remember trying to use `urljoin` here as I did refactor, and it broke things - but I can try again with test case in hand to give me assurance.